tests/test_integration.py

```
@@ -599,13 +610,91 @@ def init_with_recorded_subscription(
+            True,
+            "Home (since 11:00)",
+            "2026-08-23 12:00:00+00:00",
+            "Sample City (since 12:00)",
+        ),
+    ],
+    ids=("advanced-options", "show-time"),
+)
+async def test_async_setup_entry_renders_current_options_from_persisted_location(
+    monkeypatch: pytest.MonkeyPatch,
+    mock_hass: MagicMock,
+    show_time: bool,
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)